euth/phases/__init__.py

```diff
@@ -0,0 +1 @@
+from .contents import content
```

Comment:
Why not also importing `PhaseContent` so that apps using phase need only a single import?